.fides/systems.yml

```diff
@@ -86,3 +86,58 @@ system:
           # - fides_db.fidesuserpermissions
           # - fides_db.providedidentity
           # - fides_db.privacyrequest
+
+  # System Info
+  - fides_key: new_dictionary_system
+    name: New Dictionary System
+    organization_fides_key: default_organization
+    description: System with fields populated by dictionary.
+    system_type: System
+    administrating_department: Not defined
+    vendor_id: 34555
+    dataset_references:
+      - fides_db
+      - fides_cache
+    processes_personal_data: true
+    exempt_from_privacy_regulations: false
+    reason_for_exemption:
+    uses_profiling: true
+    legal_basis_for_profiling: Explicit consent
+    does_international_transfers: true
+    legal_basis_for_transfers: Standard contractual clauses
+    requires_data_protection_assessments: true
+    dpa_location: https://www.example.com/dpia
+    dpa_progress: pending
+    privacy_policy: https://www.example.com/privacy_policy
+    legal_name: Texas Toast
+    legal_address: 103 Trailblazing Rd. Austin, TX 29492
+    responsibility:
+      - Controller
+      - Processor
+    dpo: John Doe, CIPT
+    joint_controller_info: Jane Doe; 104 Test Road, Houston, TX 30525
```

Comment:
this is our proper `systems.yml` for Fides, should this be in a testing file?

Reply:
apologies, I thought this _was_ for testing, I'll revert back to what it was before.